service/policy/objects.proto

```diff
@@ -262,6 +307,21 @@ message KeyAccessServer {
   common.Metadata metadata = 100;
 }
 
+message Key {
+  string id = 1;
```

Comment:
```suggestion
  // the database record ID, not the key ID (`kid`)
  string id = 1;
```

Should we add a comment that this is not the `kid` but the row ID in the database?